release-notes/2024-04/README.md

```diff
@@ -0,0 +1,203 @@
+# SystemLink Enterprise release 2024-04 Release Notes
+
+The 2024-04 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
```

Comment:
Please match the text for new features as documented in https://dev.azure.com/ni/Users/_git/Documentation%20Reviews?path=/review%20output/systemlink-enterprise%20md/SystemLink%20Enterprise%20New%20Features%20and%20Changes/systemlink-enterprise-new-features-and-changes.md